src/Runtimes/Runtimes.php

```diff
@@ -52,6 +55,7 @@ public function __construct($version = '')
         $deno->addVersion('1.21', 'denoland/deno:alpine-1.21.3', 'openruntimes/deno:' . $this->version . '-1.21', [System::X86]);
         $deno->addVersion('1.24', 'denoland/deno:alpine-1.24.3', 'openruntimes/deno:' . $this->version . '-1.24', [System::X86]);
         $deno->addVersion('1.35', 'denoland/deno:alpine-1.35.2', 'openruntimes/deno:' . $this->version . '-1.35', [System::X86]);
+        $deno->addVersion('1.40', 'denoland/deno:alpine-1.40.5', 'openruntimes/deno:' . $this->version . '-1.40', [System::X86, System::ARM64]);
```

Comment:
You added support for arm64 here, on purpose?

Reply:
<img width="658" alt="Screenshot 2024-03-01 at 11 28 11" src="https://github.com/appwrite/runtimes/assets/22452787/41cf9e29-8fd0-44e6-9bed-6f85746354b0">

Yup